firmware/projects/DemoCanBar/main.cc

```diff
@@ -0,0 +1,47 @@
+/// @author Samuel Parent
+/// @date 2024-01-16
+
+#include <cstdint>
+
+#include "generated/can/can_messages.h"
+#include "generated/can/msg_registry.h"
+#include "shared/comms/can/can_bus.h"
+
+namespace bindings {
+extern shared::periph::CanBase& veh_can_base;
+extern void Initialize();
+extern void TickBlocking(uint32_t);
+}  // namespace bindings
+
+generated::can::VehMsgRegistry veh_can_registry{};
+
+shared::can::CanBus veh_can_bus{
+    bindings::veh_can_base,
+    veh_can_registry,
+};
+
+int main(void) {
+    bindings::Initialize();
+    uint32_t tick_duration = 100;
+
+    generated::can::TempSensors temp_sens_msg{};
```

Comment:
why do these need a namespace? or can we do `using namespace generated::can`?

Reply:
There will be many can messages generated and they will be based on dbc. I don't want those to pollute namespace.